ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -104,6 +104,15 @@
       + " VARCHAR SUBSTRING(str VARCHAR, pos INT, length INT), where pos is base-one indexed,"
       + " and the last argument is the length of the substring to extract.";
 
+  public static final String KSQL_WINDOWED_SESSION_KEY_LEGACY_CONFIG =
+      KSQL_CONFIG_PROPERTY_PREFIX + "windowed.session.key.legacy";
+
+  private static final String KSQL_WINDOWED_SESSION_KEY_LEGACY_DOC = ""
+      + "Version 5.1 of KSQL and earlier incorrectly excluded the end time in the record key in "
+      + "Kafka for session windowed data. Setting this value to true will make KSQL expect and "
+      + "continue to store session keys without the end time. With the default value of true "
```

Comment:
Default value is `false`! It's true for the existing queries only.

Reply:
done